setup.py

```diff
@@ -1,19 +1,17 @@
 # -*- coding: utf-8 -*-
-from setuptools import setup, find_packages
-try: # for pip >= 10
-    from pip._internal.req import parse_requirements
-except ImportError: # for pip <= 9.0.3
-    from pip.req import parse_requirements
-import re, ast
+import ast
+import re
+
+from setuptools import find_packages, setup
 
 # get version from __version__ variable in shipment_management/__init__.py
 _version_re = re.compile(r'__version__\s+=\s+(.*)')
 
 with open('shipment_management/__init__.py', 'rb') as f:
-    version = str(ast.literal_eval(_version_re.search(
-        f.read().decode('utf-8')).group(1)))
+	version = str(ast.literal_eval(_version_re.search(f.read().decode('utf-8')).group(1)))
 
-requirements = parse_requirements("requirements.txt", session="")
+with open('requirements.txt') as f:
+	install_requires = f.read().strip().split('\n')
```

Comment:
did they really remove requirement.txt parsing?
maybe it got moved to another api?